### PR TITLE
https readme update

### PR DESCRIPTION
**Pull request by idg-sam**
update readme for more detailed HTTPS instructions

### CODE REVIEW

**Review by Avery-Dunn — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -153,21 +153,27 @@ Open `application.properties` in the msal-web-sample/src/main/resources folder.
...
 - Replace *OboApi* with the API exposed in the `Web Api app` **(api://{clientId})**.
 
-If you want to use https with localhost, you must have a certificate.
-Use the `keytool` utility (included in JRE) if you want to generate self-signed certificate.
+#### HTTPS on localhost
 
-```Bash
-keytool -genkeypair -alias testCert -keyalg RSA -storetype PKCS12 -keystore keystore.p12 -storepass password
-```
+If you want your application to use https while developing on localhost, complete the instructions in this section. You may skip this step if you do not need to do this.
```
Comment (sangonzal) [inline README.md:158]: Might be good to add a note here saying that https should be used in any applications that are used in production.

**Review by sangonzal — APPROVED** [review]
If we are making this change here, it would also be good to make it to the Java web app sample to keep them in sync.

File: README.md (hunk 1 above)
Comment (idg-sam) [reply README.md:158]: How much detail should I go into?

e.g., AFAIK, when deploying to Azure, the container itself is http. Azure handles the HTTPS.

Comment (sangonzal) [reply README.md:158]: Not much, maybe would just say something along the lines:  "...You may skip this step if you do not need to do this. Note that https is essential for providing critical security and data integrity to your applications, and http should not be used outside of testing scenarios."

**Pull request merged**